Source/Core/Core/FifoPlayer/FifoAnalyzer.cpp

```diff
@@ -199,12 +200,18 @@ u32 AnalyzeCommand(const u8* data, DecodeMode mode)
   }
 
   case OpcodeDecoder::GX_CMD_CALL_DL:
-    // The recorder should have expanded display lists into the fifo stream and skipped the call to
-    // start them
-    // That is done to make it easier to track where memory is updated
-    ASSERT(false);
-    data += 8;
+  {
+    const u32 address = ReadFifo32(data);
```

Comment:
Do we want to bump the fifo version for this?  And then if the fifo version is older than the support for the DL, revert back to the previous behavior?

Reply:
Bump fifo version, yes. 

I don't really feel like reverting to the old behaviour of crashing when it encounters a display list call. Old files shouldn't have them anyway.

Reply:
If it wasn't for that single ASSERT(false) being in the read path, we wouldn't even need to bump the version number.

Reply:
Yeah, I was more thinking about the case of reading an old fifo log.  I guess we're safe with old fifos because they embedded the display list values in them?